### ISSUE TITLE
No verified contracts or information for users to externally verify

### ISSUE THREAD

**Issue by 0xdcota** [zerodevapp/kernel-7579-plugins#21]
Most of your contracts deployments go unverified 👎 . Though, this understandable given the hassle of all chains. 

However, if you are not going to verify them at least provide definition of the solidity versions you use in your deployments in the `foundry.toml` for devs to verify as they need on the fly. 

https://github.com/zerodevapp/kernel-7579-plugins/blob/facca7ce1770afeb1489d9a1d2c49f7fbe2f9255/actions/recovery/foundry.toml#L5

Refer to the list of deployments [here](https://github.com/zerodevapp/kernel-7579-plugins?tab=readme-ov-file#addresses).

**Comment by leekt** [comment]
Good point, thanks for pointing this out, and thanks for understanding that this is painful when it comes to multichain. We are working on stuffs to make sure we don't miss this out on every plugins/deployments, will add this support as soon as possible.

**Comment by leekt** [comment]
Not going to close this issue until we figure out stuff properly
